Tools/patchcheck/patchcheck.py

```diff
@@ -147,85 +134,11 @@ def changed_files(base_branch=None):
     else:
         sys.exit('need a git checkout to get modified files')
 
-    filenames2 = []
-    for filename in filenames:
-        # Normalize the path to be able to match using .startswith()
-        filename = os.path.normpath(filename)
-        if any(filename.startswith(path) for path in EXCLUDE_DIRS):
-            # Exclude the file
-            continue
-        filenames2.append(filename)
-
+    # Normalize the path to be able to match using .startswith()
```

Comment:
Comment is not accurate

Reply:
perhaps?
```suggestion
    # Normalize all paths in 'filenames' to be able to match using .startswith()
```

Reply:
Oh my comment was in reaction to the mention of startswith, which is used in the deleted code but not anymore.
But you’re right that there’s also a number issue now.